packages/core/useWindowSize/index.ts

```diff
@@ -71,6 +76,10 @@ export function useWindowSize(options: UseWindowSizeOptions = {}) {
   tryOnMounted(update)
   useEventListener('resize', update, { passive: true })
 
+  if (window && type === 'visual') {
```

Comment:
```suggestion
  if (window && type === 'visual' && window.visualViewport) {
```

Should we also check this?

Reply:
yeah, It's needed! I thought that the `useEventListener ` has made a null value judgment internally😂